src/components/Select/components/SelectMenu/SelectMenu.style.tsx

```diff
@@ -25,13 +25,33 @@ export const optionStyle = ({
   `;
 };
 
-export const menuStyle = ({ status, size }: Props) => (theme: Theme): SerializedStyles => css`
+/** we should search for an alternative for this function.... */
+const getHeight = ({ items, size }: { items: number; size: 'md' | 'sm' | undefined }) => {
```

Comment:
not sure i get 100% what are the metrics here? Can we have a better description of what this function calculates? Specifically the `46`, `277`, `265`